dapr/clients/grpc/client.py

```diff
@@ -459,6 +460,73 @@ def get_bulk_state(
             items=items,
             headers=call.initial_metadata())
 
+    def query_state_alpha1(
```

Comment:
We had a chat about this offline. By appending alpha1(), we're basically forcing the user to do a code migration when this feature goes out of preview. In the ideal case, we'd hope that there aren't breaking changes from alpha1 -> release. In this scenario we'd have unnecessarily forced early adopters to change their code once the feature became release.

I propose we call the interface query_state, and simply include a warn() that this is an alpha feature, which we can simply remove when the final API goes to release, provided there are no changes

Reply:
I had voiced same concerns with regards to go-sdk and other SDKs, but unfortunately wasn't able to persuade the maintainers.